guides/v2.3/install-gde/prereq/nginx.md

```diff
@@ -214,66 +258,93 @@ Magento requires several [PHP extensions](php-centos-ubuntu.html) to function pr
 
 1. Install `php-fpm`:
 
-		yum -y install php70w-fpm
+    ```bash
+    yum -y install php70w-fpm
+    ```
 
 2. Open the `/etc/php.ini` file in an editor.
 
 3. 	Uncomment the `cgi.fix_pathinfo` line and change the value to `0`.
 
 4. Edit the file to match the following lines:
 
-		memory_limit = 2G
-		max_execution_time = 1800
-		zlib.output_compression = On
+    ```conf
+    memory_limit = 2G
+    max_execution_time = 1800
+    zlib.output_compression = On
+    ```
 
 {:.bs-callout .bs-callout-info}
 We recommend setting the memory limit to 2G when testing Magento. Refer to [Required PHP settings]({{page.baseurl }}/install-gde/prereq/php-settings.html) for more information.
 
 5. Uncomment the session path directory and set the path:
 
-		session.save_path = "/var/lib/php/session"
+    ```conf
+    session.save_path = "/var/lib/php/session"
+    ```
 
 6. Save and exit the editor.
 
 7. Open `/etc/php-fpm.d/www.conf` in an editor.
 
 8. Edit the file to match the following lines:
 
-		user = nginx
-		group = nginx
-		listen = /run/php-fpm/php-fpm.sock
-		listen.owner = nginx
-		listen.group = nginx
-		listen.mode = 0660
+    ```conf
+    user = nginx
+    group = nginx
+    listen = /run/php-fpm/php-fpm.sock
+    listen.owner = nginx
+    listen.group = nginx
+    listen.mode = 0660
+    ```
 
 9. Uncomment the environment lines:
 
-		env[HOSTNAME] = $HOSTNAME
-		env[PATH] = /usr/local/bin:/usr/bin:/bin
-		env[TMP] = /tmp
-		env[TMPDIR] = /tmp
-		env[TEMP] = /tmp
+    ```cond
```

Comment:
cond? or conf?

Reply:
Agh... it's supposed to be `conf`. Thanks!